### DISCUSSION TITLE
Shadowing RDP sessions

### DISCUSSION THREAD

**Discussion by techmonpiano** [Ylianst/MeshCentral#3940] [Q&A]
Greetings,

Currently, with MeshCentral one can shadow RDP sessions on a Windows Server.  Thanks for this great feature, Ylianst and Team!

We have more than one IT contact here and sometimes multiple of us want to shadow the same RDP session simultaneously.

It seems, however, that it does not work for multiple simultaneous shadowing of the same RDP session.  Is this correct?
(At least for our Windows 2012 R2 instance)

Thanks for your time :)
Shawn

**Comment by techmonpiano** [comment]
Oh, I apologize 
I just tested again and it is working fine for me now with multiple simultaneous shadowing of a single RDP session. :)

Will be quite nice if the ability to shadow RDP sessions also is added to MeshCentral Router app.  (Currently one can only remote to console I think? if using MeshCentral Router)

Thanks again!

**Comment by Ylianst** [comment]
One issue we have is that the MeshAgent only allows remote access to one RDP session. So, if the first user connects and selects RDP session 1, the second user can select any RDP session whey want, they will still see session1. Bryan is working on adding support for multiple desktop sessions at the same time, not sure when that will be released.

**Comment by techmonpiano** [comment]
Thank you Ylianst!

**Comment by MeowLucian** [comment]
Dear @Ylianst 

Will this shadow mode still be implemented in plan ?

I have a request that using shadow mode, the screen will not be logout and the user who sits right in front the controlled PC can still control it. Just like TeamViewer.

Thanks
Lucian

**Reply by si458** [reply]
I believe when u click connect, we already check for rdp sessions, and list them, then u can click which one to control/watch

To make sure u don't control that session, just untick the INPUT button at the bottom of the remote tab

Is this what you mean?
